_posts/2025-02-16-19.0.1-release.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: "Apache Arrow 19.0.1 Release"
+date: "2025-02-16 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow team is pleased to announce the 19.0.1 release.
+
+This release primarily addresses a bug in the recent Arrow 19.0.0 release which
+prevents Arrow C++ and libraries binding it (e.g., Python, R) from reading
+Parquet files created by Arrow Rust v53.0.0 or more recent. See the [19.0.0
+release blog post][4] for more information.
+
+This release includes [**12 resolved issues**][1] from [**9 distinct
+contributors**][2]. See the [Install Page][5] to learn how to get the libraries for
+your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## C++ notes
+
```

Comment:
```suggestion

- A bug has been fixed which caused reading of Parquet files written by Arrow Rust v53.0.0 or higher to always fail [(#45283)](https://github.com/apache/arrow/issues/45283).
- A workaround has been added to increase compatibility with newer versions of the AWS C++ SDK ([#45304](https://github.com/apache/arrow/issues/45304)).
- An overflow bug has been fixed which could cause segmentation faults when joining inputs over a certain size ([#44513](https://github.com/apache/arrow/issues/44513)).
```

Reply:
cc @pitrou @zeroshade for a review on the C++ notes

Reply:
LGTM